#! /usr/bin/env python
# Create a cfg file for getting control parameters for dmoc optimal controller

PACKAGE = 'gcop_ctrl'

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("iterate"        , bool_t   , 0                 , "iterate"     , False)
#gen.add("enable_debug"   , bool_t   , 0                 , "enable debug", True)
#gen.add("Nit"             , int_t    , 0                 , "Number of iterations of dmoc" , 30     , 0   , 1000)
#gen.add("zf"             , double_t    , 0                 , "Goal z" , 2     , 1   , 20)

exit(gen.generate(PACKAGE , "Config" , "QRotorIDModelInterface"))




